vivisect/tests/testvivisect.py

```diff
@@ -186,10 +186,10 @@ def test_basic_apis(self):
         self.assertTrue(len(vw.getLocations()) > 76000)
 
         # tuples are Name, Number of Locations, Size in bytes, Percentage of space
-        ans = {0: ('Undefined', 0, 53840, 14),
-               1: ('Num/Int',   727, 3922, 1),
-               2: ('String',    256, 6448, 1),
-               3: ('Unicode',   172, 5530, 1),
+        ans = {0: ('Undefined', 0, 53924, 14),
```

Comment:
do we know what changed to make these differ?

Reply:
Some numbers/integer locations became string locations. 
Total byte discovery is roughly the same, just more strings this time.